exchange/bidder_test.go

```diff
@@ -114,6 +114,67 @@ func TestSingleBidder(t *testing.T) {
 	}
 }
 
+// TestMultiBidder makes sure all the requests get sent, and the responses processed.
+func TestSingleBidderModifiesTestFlagValue(t *testing.T) {
+	respStatus := 200
+	respBody := "{\"bid\":false}"
+	server := httptest.NewServer(mockHandler(respStatus, "getBody", respBody))
+	defer server.Close()
+
+	requestHeaders := http.Header{}
+	requestHeaders.Add("Content-Type", "application/json")
+
+	bidAdjustment := 2.0
+	firstInitialPrice := 3.0
+	secondInitialPrice := 4.0
+	mockBidderResponse := &adapters.BidderResponse{
```

Comment:
I think we have an opportunity here to extract a simple bid response and use that in this test and other tests rather than creating a new bid response for each test function. This will reduce redundant code for defining similar bid responses

Reply:
This test case is no longer needed in PR #1387